earth_enterprise/src/third_party/gtest/SConscript

```diff
@@ -22,11 +22,13 @@ buildGtest = False
 
 # Check environment for libgtest.so
 conf = Configure(env)
-if not conf.CheckLib('gtest'):
-    buildGtest = True
-
 env = conf.Finish()
 
+# If building release, don't include gtest so scons won't add it as a dependency.
+if not env['release']:
+    if not conf.CheckLib('gtest'):
```

Comment:
I'll check it tomorrow, but this solution may only work for RHEL/CentOS. In Ubuntu, it never builds GTest, but still links it.